app/forms/registration_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class RegistrationForm < ApplicationForm
+  def save(create_params)
+    return false unless validate_client(create_params[:client_id])
+
+    user = User.new(create_params.except(:client_id))
+
+    unless user.valid?
```

Comment:
```suggestion
    if user.invalid?
```

Rails rails rails xD